javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+      if(this.currentTime > 3) {
```

Comment:
Justo esto es lo que esperamos que haga la funcion pero tenemos que definilo con el callback que esta en los parametros al final del dia este callback puede ser el clearInterval o clearTimeout 👀 
```suggestion
      if(callback) callback()
```